aten/src/ATen/native/quantized/cpu/q_adaavgpool.cpp

```diff
@@ -227,11 +229,64 @@ Tensor q_adaptive_avg_pool2d(const Tensor& input, IntArrayRef output_size) {
     return output;
   }
 }
+
+#ifdef USE_PYTORCH_QNNPACK
+Tensor qnnpack_adaptive_avg_pool2d(
```

Comment:
This looks like it's in the wrong diff.

Reply:
Yes. This was a stacked diff in phabricator. Does not get exported as such. I needed to do `ghexport`. Ok I will address you comments and then do ghexport for the entire stack. Dont review any other diffs till then.